acceptance-tests/build.gradle

```diff
@@ -10,5 +10,6 @@ dependencies {
   testFixturesImplementation 'org.apache.tuweni:tuweni-bytes'
   testFixturesImplementation 'org.apache.tuweni:tuweni-toml'
   testFixturesImplementation 'org.junit.jupiter:junit-jupiter-api'
-  testFixturesImplementation "org.testcontainers:testcontainers"
+  testFixturesImplementation 'org.testcontainers:testcontainers'
+  testFixturesImplementation 'org.yaml:snakeyaml'
```

Comment:
Why snakeyaml? We just use Jackson to parse yaml in other places? For example in `YamlValidatorKeyProvider`.

Reply:
👍 Reverting to Jackson YAMLFactory implementation.